src/components/PreviewCompatibleImage.js

```diff
@@ -16,8 +16,13 @@ const PreviewCompatibleImage = ({ imageInfo }) => {
     return <Img style={imageStyle} fluid={childImageSharp.fluid} alt={alt} />
   }
 
-  if (!!image && typeof image === 'string')
+  if (!!image && typeof image === 'string') {
     return <img style={imageStyle} src={image} alt={alt} />
+  }
+
+  if (typeof imageInfo === 'string') {
+    return <img style={imageStyle} src={imageInfo} alt={alt} />
```

Comment:
The fact that you need this has me worried. Makes me think that we're not processing images correctly for lunr (i.e. not tracking the correctly processed ones).

Reply:
For now, I think I'd like to remove this section, just to make sure we're passing the `childImageSharp` property correctly.